test/dhtest.c

```diff
@@ -198,7 +208,6 @@ static int dh_test(void)
     BN_free(cpriv_key);
     BN_GENCB_free(_cb);
     DH_free(dh);
-
```

Comment:
I added another fixup to restore this empty line which has been deleted unnecessarily. But I didn't push it yet in order not to interrupt the ci.